content/pt-br/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -2,43 +2,41 @@
 title: Sobrecarga de Pod 
 content_type: concept
 weight: 50
+update_date: 2022-07-22
+origin_version: 1.24
+contributors:
+- DonatoHorn
+reviewers:
```

Comment:
I'm really not keen on seeing these lines added. They aren't a convention that we've accepted within SIG Docs.

Reply:
Sorry, understand.